content/departments/product-engineering/engineering/enablement/dev-experience/index.md

```diff
@@ -38,9 +38,9 @@ The Dev Experience team, or DevX for short, is a team focused on improving the d
 
 ## Contact
 
-- **Discussions and questions**: #dev-experience channel and [developer experience GitHub discussions](https://github.com/sourcegraph/sourcegraph/discussions/categories/developer-experience)
+- **Discussions and questions**: [#dev-experience](https://sourcegraph.slack.com/archives/C01N83PS4TU) channel and [developer experience GitHub discussions](https://github.com/sourcegraph/sourcegraph/discussions/categories/developer-experience)
```

Comment:
Slack links, e.g. `#dev-experience`, are automatically generated by the handbook: https://handbook.sourcegraph.com/departments/product-engineering/engineering/enablement/dev-experience#contact

Reply:
TIL! Thank you, I'll revert that :)